com.salesforce.b2eclipse.jdt.ls/src/main/java/com/salesforce/b2eclipse/classpath/BazelClasspathContainer.java

```diff
@@ -117,10 +129,74 @@ public static void clean() {
         }
     }
 
+    private IClasspathEntry computeFilePathForRunnerJar(WorkProgressMonitor progressMonitor) {
+        final String runnerJarName = "Runner_deploy-ijar.jar";
+
+        BazelWorkspaceCommandRunner bazelWorkspaceCmdRunner = getBazelCommandManager()
+                .getWorkspaceCommandRunner(getBazelWorkspaceRootDirectory());
+        File bazelBinDir = bazelWorkspaceCmdRunner.getBazelWorkspaceBin(progressMonitor);
+        File testRunnerDir = new File(bazelBinDir, IMPLICIT_RUNNER.replaceAll("/", File.separator));
+
+        if (!testRunnerDir.exists()) {
+            return null;
+        }
+
+        String javaToolName = "remote_java_tools_" + bazelWorkspaceCmdRunner.getOperatingSystemDirectoryName();
+
+        String javaToolsPath = javaToolName + File.separator + "java_tools";
+        File javaToolsDir = new File(testRunnerDir, javaToolsPath);
+        if (!javaToolsDir.exists()) {
+            return null;
+        }
+        File runnerJar = new File(javaToolsDir, runnerJarName);
+        if (!runnerJar.exists()) {
+            return null;
+        }
+        return getJavaCoreHelper().newLibraryEntry(fromOSString(runnerJar.toString()), null, null);
+    }
+
+    private IClasspathEntry jarsToClasspathEntry(BazelWorkspaceCommandRunner bazelCommandRunner,
+            WorkProgressMonitor progressMonitor, AspectOutputJars jarSet) {
+        IClasspathEntry cpEntry = null;
+        File bazelOutputBase = bazelCommandRunner.getBazelWorkspaceOutputBase(progressMonitor);
+        File bazelExecRoot = bazelCommandRunner.getBazelWorkspaceExecRoot(progressMonitor);
+        IPath jarPath = getJarPathOnDisk(bazelOutputBase, bazelExecRoot, jarSet.getJar());
+        if (jarPath != null) {
+            IPath srcJarPath = getJarPathOnDisk(bazelOutputBase, bazelExecRoot, jarSet.getSrcJar());
+            IPath srcJarRootPath = null;
+            cpEntry = getJavaCoreHelper().newLibraryEntry(jarPath, srcJarPath, srcJarRootPath);
+        }
+        return cpEntry;
+    }
+
+    private List<IClasspathEntry> calculateBazelClasspath(Collection<AspectPackageInfo> aspectPackageInfos,
+            WorkProgressMonitor progressMonitor) {
+        BazelWorkspaceCommandRunner bazelWorkspaceCmdRunner = getBazelCommandManager()
+                .getWorkspaceCommandRunner(getBazelWorkspaceRootDirectory());
+        // filter-out main modules and
+        List<IClasspathEntry> classpathEntries = aspectPackageInfos.stream() //
+                .filter((AspectPackageInfo info) -> ObjectUtils.notEqual("java_library", info.getKind())
```

Comment:
We could store these constants in a static collection (e.g. `Set`) and shorten this to:
`!COLLECTION_NAME.contains(info.getKind())`